app/views/conversations/index.html.erb

```diff
@@ -0,0 +1,20 @@
+<div class=”ui segment”>
+ <h3>Mailbox</h3>
+ <div class=”ui list”>
+  <div class=”item”>
+   <% @conversations.each do |conversation| %>
+   <% if conversation.sender_id == current_user.id ||      conversation.recipient_id == current_user.id %>
```

Comment:
I'd put this restriction in the controller - only load convs that the current user should see.